app/static/src/app/components/run/RunTab.vue

```diff
@@ -25,15 +25,19 @@ export default defineComponent({
     },
     setup() {
         const store = useStore();
-        const canRunModel = computed(() => !!store.state.model.odinRunner && !!store.state.model.odin);
+
+        const requiredAction = computed(() => store.state.model.requiredAction);
```

Comment:
```suggestion
        const requiredAction = reactive(computed(() => store.state.model.requiredAction));
```

Please feel free to ignore my suggestion. I just don't like the fact that one has to use requiredAction.value every time.

Reply:
Yeah reactive only works for object. I think for primitives, one could explore add $ sign to ref `$ref()`.